### DISCUSSION TITLE
委托

### DISCUSSION THREAD

**Discussion by Kujojolen** [share121/inter-knot#1451] [其他]
各位绳匠帮忙求解！！！
![Uploading IMG_20240809_145903.jpg…]()

**Comment by lllliufeng** [comment]
没传完哦~

**Reply by lllliufeng** [reply]
uploading变成文件名后才有用哦~

**Reply by Kujojolen** [reply]
![IMG_20240809_145903](https://github.com/user-attachments/assets/c420ed49-0725-45e4-b610-f15d7d37e7c8)
这个

**Comment by Kujojolen** [comment]
![IMG_20240809_145903](https://github.com/user-attachments/assets/1b641d97-7e37-463d-b7e3-80e683e360af)

**Comment by lllliufeng** [comment]
9500Ω 90000Ω

**Reply by lllliufeng** [reply]
考察电表的改装，串联时电流一定，满偏时电流为1mA，满偏电压U<sub>g</sub>=I<sub>g</sub>*R<sub>g</sub>=0.5V
由欧姆定律，且U<sub>AB</sub>=10V
U<sub>AB</sub>/R<sub>AB</sub>=U<sub>g</sub>/R<sub>g</sub>
解得R<sub>AB</sub>=10000Ω，R<sub>1</sub>=R<sub>AB</sub>-R<sub>G</sub>=9500Ω
由欧姆定律，且U<sub>AC</sub>=100V
U<sub>AB</sub>/R<sub>AB</sub>=U<sub>AC</sub>/R<sub>AC</sub>
解得R<sub>AC</sub>=100000Ω，R<sub>2</sub>=R<sub>AC</sub>-R<sub>AB</sub>=90000Ω
